models/alumni.js

```diff
@@ -1,4 +1,5 @@
 const mongoose = require("mongoose")
+const Schema = mongoose.Schema;
 
 const alumniSchema = new mongoose.Schema({
```

Comment:
since you imported Schema above, you can use that here:
```
const alumniSchema = new Schema({
```

Reply:
changed.